app/javascript/hooks/queries/rooms/useRooms.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import {useQuery} from "react-query";
+import axios from "axios";
+
+export default function useRooms() {
+  return useQuery("getRooms", () =>
+    axios.get('/api/v1/rooms.json', {
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+      }
+    }).then(resp => resp.data.data)
```

Comment:
I think it's much cleaner to use `Async/Await` syntax when it's possible.
That doesn't mean that we won't be using the `then/catch` syntax.